harness/determined/common/api/bindings.py

```diff
@@ -726,8 +726,8 @@ class trialv1Trial(Printable):
     checkpointCount: "typing.Optional[int]" = None
     endTime: "typing.Optional[str]" = None
     latestValidation: "typing.Optional[v1MetricsWorkload]" = None
+    logPolicyMatched: "typing.Optional[str]" = None
     logRetentionDays: "typing.Optional[int]" = None
-    logSignal: "typing.Optional[str]" = None
```

Comment:
Is this going to cause a lot of breakages in the python sdk?

This might be a place to deprecate and stop using the old field, and introduce the new field as an addition.

Reply:
Looks like it has to be in this PR? React Tests in CI failed.

Reply:
I think the right way to test it would be to run a 0.37.0 python sdk script that fetches a trial from a master that has your change.  I think that's the case I'm more concerned about.

As far as the webui goes, you'll probably want to fix the name in this same PR (regardless of if you break the API or just append a new field).

Personally, I would not break the API at all, I'd just leave the field as deprecated and unused.  It's not like we never break our API but it doesn't seem worth it here.

Reply:
I found log_signal was merged into main after 0.37.0 release cut-off. log_signal was merged into main on 09/25 and 0.37.0 cut-off was on 09/23.